Cranky.toml

```diff
@@ -148,6 +148,7 @@ warn = [
 ]
 
 allow = [
+  "clippy::comparison_chain",      # annoying
```

Comment:
protesting this one a little bit: helped me out a couple of times not seeing possible match expressions which looked better indeed.
The examples shown on the doc page are fairly convincing. https://rust-lang.github.io/rust-clippy/master/#/clippy::comparison_chain
Would a local ignore do the job just as well?